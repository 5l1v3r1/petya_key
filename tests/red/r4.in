ceNQvr2pYRvJ4D1oELoK4z6DVnM11HHZiQomqY7fNXQDAvAMhkkyzvzBWw2CzCbdSM2HuJPYENrnTNDyEFSVrb64f1
