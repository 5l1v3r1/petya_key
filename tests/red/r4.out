i8kV3MUF8kiwR31j
